libcodechecker/util.py

```diff
@@ -313,3 +313,10 @@ def sizeof_fmt(num, suffix='B'):
             return "%3.1f%s%s" % (num, unit, suffix)
         num /= 1024.0
     return "%.1f%s%s" % (num, 'Yi', suffix)
+
+
+def escape_like(string, escape_char='*'):
+    """Escape the string parameter used in SQL LIKE expressions."""
+    return string.replace(escape_char, escape_char * 2) \
```

Comment:
So a `*` becomes `**`? How does this affect the query?

If I have `csordas*marton` as filter and I have `csordas*marton` and `csordasFOOBARmarton` as run names in the database I get only the first as result?

Reply:
If you have two run names (`csordas*marton` and `csordasFOOBARmarton`) you get only the first as result for `csordas*marton` filter.

Reply:
This escaping applies to the run and product names when the user starts typing an infix of these for filtering. I think it is not necessary to accept joker characters in this situation, so it wouldn't be a problem if asterisk is interpreted as an exact character.